### PR TITLE
Bug fix reports

### PR DESCRIPTION
**Pull request by dfalster**
Addressing #593

### CODE REVIEW

**Comment by dfalster** [comment]
Fixing #593. 

Building of reports is now working again. Also requires some changes in austraits package in  `austraits::plot_trait_distribution_beeswarm` and `extract_trait`. (see changes at https://github.com/traitecoevo/austraits/tree/new_format)


For now these can be installed with 

```
remotes::install_github("traitecoevo/austraits@ new_format", quick=TRUE)
```

Also renamed 

- trait_definitions -> definitions (consistent with previous structures in austraits)
- renamed `build_dataset_report` -> `dataset_generate_report` (as per #590 )

You'll need to rebuild all of austraits though, sorry. To clear the case, use

```
remake::make("tidy")
```

File: man/dataset_generate_reports.Rd [hunk 1]
```diff
@@ -1,22 +0,0 @@
-% Generated by roxygen2: do not edit by hand
```
Comment (ehwenk) [inline man/dataset_generate_reports.Rd]: Confirming that there is no longer an option to build reports for all studies automatically? Instead each dataset needs to be explicitly listed using the function `dataset_generate_report`. (I don't think this is a problem - and is probably very sensible as the number of datasets gets ever larger, but I'd previously occasionally rebuild all reports to have current copies.)

Comment (dfalster) [reply man/dataset_generate_reports.Rd]: Actually you can, just not a seperate function. Just add multiple values for `dataset_id`. For example

```
# 2 studies
dataset_gerenerate_report(c("Angevin_2011", "Bragg_2002", austraits)

# all studies
dataset_gerenerate_report(dirs("data"), austraits)
```

Comment (ehwenk) [reply man/dataset_generate_reports.Rd]: Excellent. Good to be reminded of the ```dirs("data")``` code.

**Pull request merged**